src/service/index.js

```diff
@@ -3,4 +3,5 @@ import { addItemToLikedVideos, deleteItemFromLikedVideos } from './like-service'
 import { getAllCategories } from './category-service';
 import { loginUser, signupUser } from './auth-service';
 import { getWatchLaterItems, addItemToWatchLater, removeItemFromWatchLater } from './watch-later-service';
-export { getAllVideos, getVideo, getAllCategories, loginUser, signupUser, addItemToLikedVideos, deleteItemFromLikedVideos, addItemToWatchLater, removeItemFromWatchLater,getWatchLaterItems };
\ No newline at end of file
+import { getHistoryVideos, addVideoToHistory, removeVideoFromHistory, removeAllVideoFromHistory } from './history-service';
+export { getAllVideos, getVideo, getAllCategories, loginUser, signupUser, addItemToLikedVideos, deleteItemFromLikedVideos, addItemToWatchLater, removeItemFromWatchLater,getWatchLaterItems, getHistoryVideos, addVideoToHistory, removeVideoFromHistory, removeAllVideoFromHistory };
```

Comment:
why import and then export when you can export it directly by writing `export { function1, function2 } from "path/to/file"`